content/scss/_c-menu.scss

```diff
@@ -10,39 +10,41 @@
 $g-c-menu-z: 60 !default;
 
 /* Component
-   ========================================================================== */
+      ========================================================================== */
```

Comment:
General comment about the dropdowns (this may be related to the changes here, or not ; I haven't compared with the previous revision):

- When the button is clicked and the items are visible (unfolded), the button still changes on mouseover. Since clicking outside the unfolded items, on or outside the button, has the same behavior (i.e. collapse the items), I think it should react on mouseover.
- Have a look at this screenshot with the icons variant:
![screenshot](https://user-images.githubusercontent.com/45588452/130516062-f487e201-e1ce-4644-99fa-076ea28fda48.png)

Reply:
Sorry about the icons layout, I fixed it this morning.
I will check the mouseover change with @Wolfr!

Reply:
Oh, b.t.w. my sentence above should be "should _not_ react on mouseover".

Reply:
What's up with the fonts on your screenshot? @thusc ? Didn't we include Inter locally so everyone should see Inter?

Reply:
I had a quick look and indeed I don't think "Inter UI" (I think that's the name in your CSS) is loaded. I have no idea why TeX Gyre Heros shows up though (I'm pretty sure DejaVu Sans Mono is some default within Firefox / Linux).